src/RoboFile/RoboFileInterface.php

```diff
@@ -43,4 +43,26 @@ public function runTestSuites(
 	 * @since   3.7.0
 	 */
 	public function prepareTestingPackage($params = array('dev' => false));
+	/**
+	 * Function for actual execution of a individual test of this extension
+	 *
+	 * @param   array  $opts  Array of configuration options:
+	 *                        - 'env': set a specific environment to get configuration from
+	 *                        - 'debug': executes codeception tasks with extended debug
+	 * 						  - 'suite':
+	 * 						  - 'test': example: 'install/InstallWeblinksCest.php/installWeblinks'
+	 *
+	 * @return void
+	 *
+	 * @since   x.x.x
+	 */
+	public function runTestSingle(
```

Comment:
ok, so we make it a separate robo command in the parent class, instead of it being in the interface. But, in this way, we need to alter the `runTestSuites` implementation in the weblinks repository, so we check there if the whole suite is needed to run, or an independent test.